genericjpa/db/src/main/java/org/hibernate/search/annotations/UpdateInfo.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.annotations;
+
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * @author Martin Braun
+ * @hsearch.experimental
+ */
+@Target({FIELD, METHOD, TYPE})
+@Retention(RUNTIME)
+@Repeatable(UpdateInfos.class)
+public @interface UpdateInfo {
```

Comment:
Yes, we had it before, and as I said back then, I just think this approach is not really what a user would expect or be willing to do. It's just too much work on their side.

Also it's not clear to me what benefit these annotations provide at all: Their info can be retrieved from the model itself: table name, id type etc. Why asking the user to express this info a second time?